setuptools/tests/test_build_meta.py

```diff
@@ -157,6 +157,38 @@ def test_build_wheel(self, build_backend):
 
         assert os.path.isfile(os.path.join(dist_dir, wheel_name))
 
+    @pytest.mark.xfail(reason="Known error, see GH #1671")
+    def test_build_wheel_with_existing_wheel_file_present(self, tmpdir_cwd):
+        # Building a wheel should still succeed if there's already a wheel
+        # in the wheel directory
+        files = {
+            'setup.py': "from setuptools import setup\nsetup()",
+            'VERSION': "0.0.1",
+            'setup.cfg': "[metadata]\nname = foo\nversion = file: VERSION",
+            'pyproject.toml': DALS("""
+                [build-system]
+                requires = ["setuptools", "wheel"]
+                build-backend = "setuptools.build_meta
+            """),
+        }
+        build_files(files, prefix=str(tmpdir_cwd))
+
+        dist_dir = os.path.abspath('pip-wheel-preexisting')
+        os.makedirs(dist_dir)
+
+        # make first wheel
+        wheel_one = self.get_build_backend().build_wheel(dist_dir)
+
+        # change version
+        with open("VERSION", "wt") as version_file:
+            version_file.write("0.0.2")
+
+        # make *second* wheel
+        wheel_two = self.get_build_backend().build_wheel(dist_dir)
+
+        assert os.path.isfile(os.path.join(dist_dir, wheel_one))
+        assert wheel_one != wheel_two
```

Comment:
Just tested, this actually fails at this assertion, not with the `ValueError` we are expecting.

Reply:
I'm not quite sure what's going on, but I think it's either not picking up the `setup.cfg` or it's not using it, because I'm seeing it generate wheels like `'UNKNOWN-0.0.0-py3-none-any.whl'`.

Reply:
The original test (prior to force push) seems to work fine.

Reply:
Ah, found the problem, it was the `prefix=tmpdir_cwd` part. I've removed that and it now fails as expected.